source/core/src/main/com/csse3200/game/areas/terrain/TerrainResource.java

```diff
@@ -92,6 +93,22 @@ public TerrainResource(MapHandler.MapType mapType) {
         this.setPossibleTiles();
     }
 
+    public int getForestSize() {
```

Comment:
I think it's reasonable to combine these four functions to just take in an enum for the tile/map type